index.html

```diff
@@ -98,12 +98,69 @@ <h5 class="s1__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, se
 
     </section1>
 
-    <div class="statistic">
+    <section2 class="s2">
+
         <div class="container">
+            
+            <div class="s2__inner">
+                <h3 class="s2__suptitle">We work with</h3>
+                <h1 class="s2__title">Amazing services</h1>
+            </div>
+
+            <div class="services1">
+
+                <div class="s2__images">
+                    <div class="serv">
+                        <img src="assets/images/ALARM.png" alt="1">
+                    </div>
+                    <div class="serv">
+                        <h3>Photography</h3>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
+                    </div>
+
+                    <div class="serv">
+                        <img src="assets/images/LINE GRAPH.png" alt="2">
+                    </div>
+                    <div class="serv">
+                        <h3>Web design</h3>Ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
+                    </div>
+
+                    <div class="serv">
+                        <img src="assets/images/COMPUTER _ OK.png" alt="3">
+                    </div>
+                    <div class="serv">
+                        <h3>Creativity</h3>Dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
+                    </div>
```

Comment:
You should combine img and text inside one div, not separated.

Reply:
I tried so. But then the text is placed below the picture, and should be placed next to the picture.